modules/wri_subpage/wri_subpage.module

```diff
@@ -21,43 +21,47 @@ function wri_subpage_node_insert(EntityInterface $entity) {
  */
 function wri_subpage_node_update(EntityInterface $entity) {
   if (isset($entity->field_parent_page)) {
-    $parent = $entity->field_parent_page->entity;
-    if ($parent && $parent->id()) {
-      // Set the correct menu for microsites vs all others.
-      'microsite' == $parent->bundle() ? $menu_name = 'microsites' : $menu_name = 'page-hierarchies';
-      $menu_link_manager = \Drupal::service('plugin.manager.menu.link');
+    $parentPage = $entity->field_parent_page->getValue();
+    $oriParentPage = $entity->original->field_parent_page->getValue();
```

Comment:
You'll want to make sure `$entity->original` exists before running this, because a new subpage throws an error right now. If there is no value in `$entity->original`, go into the if statement below this line.

Reply:
Fixed!